CreeDictionary/CreeDictionary/settings.py

```diff
@@ -169,12 +155,10 @@
 
 ############################## staticfiles app ###############################
 
-if RUNNING_ON_SAPIR:  # pragma: no cover
-    # on sapir /cree-dictionary/ is used to identify the service of the app
-    # XXX: this is kind of a hack :/
-    STATIC_URL = "/cree-dictionary/static/"
-else:
+if DEBUG:
     STATIC_URL = "/static/"
+else:
+    STATIC_URL = "http://localhost:8001/"
```

Comment:
in production, static files are served by python built-in module `http` thru port 8001 inside the docker container